src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__wrapper calendar__wrapper--start-day--mon calendar__wrapper--month-length-28">
```

Comment:
calendar__wrapper--month-length-28   <--- you forgot to add "-" before 28, thats why it didn't work.

calendar__wrapper--month-length--28 <-- thats correct name